CHANGES.txt

```diff
@@ -21,6 +21,9 @@ Changes
 Fixes
 =====
 
+- Fixed issue that caused closed partitions to show a ``CRITICAL`` data state
+  in the Admin UI.
```

Comment:
could also use `resulted in` in place of `caused` to avoid the caused closed ... :)